src/components/WorkPositionDetails.vue

```diff
@@ -2,19 +2,26 @@
   <div>
     <h1>Position details</h1>
     <div>
-      Name:
-      <input v-model="name" :disabled="disabled == 1">
+     <md-field>
+       <label>Name</label>
+       <md-input v-model="name" type="text" :disabled="disabled == 1"/>
```

Comment:
Dlaczego diabled jest intem zamiast booleanem?

Reply:
Bo tak było w jakimś przykładzie w internetach. Można by to zmienić, ale to by się wiązało z modyfikacją szczegółów użytkownika też, bo tam jest tak samo to zrobione.

Reply:
No to zacznijmy od tej klasy ;)